anime_downloader/util.py

```diff
@@ -459,3 +459,84 @@ def type_cast_value(self, ctx, value):
             return ast.literal_eval(value)
         except:
             raise click.BadParameter(value)
+
+
+class Process:
+    def __init__(self, name, cmdline, pid):
+        self.name = name
+        self.pid = pid
+        self.cmdline = cmdline
+
+    def __str__(self):
+        return str({
+            'name': self.name,
+            'pid': self.pid,
+            'cmdline': self.cmdline
+        })
+
+
+def getAllProcesses_Win32():
+    placeholder = list()
+    out = os.popen('WMIC path win32_process get Caption,Processid,Commandline').read(
+    ).split('\n')[::2][1:]
+    for line in out:
+        f = line.split()
+        if bool(f):
```

Comment:
Unnecessary bool cast - `if f:` should suffice. Also, consider naming the variable something meaningful